### DISCUSSION TITLE
Citing websites, 'Accessed' date missing

### DISCUSSION THREAD

**Discussion by SarahJMills** [retorquere/zotero-better-bibtex#2845] [Q&A]
Hi

I'm using Zotero 6.0.37 with BibTeX to create a book project in Quarto (in RStudio). My University requires Harvard style referencing. At the moment, when citing websites, both in-text citations and the end-text reference list are missing the 'Accessed' field (in-text appear as 'Author, no date'...end-text as 'Author (no date)'). NB the Accessed field is definitely populated in the source Zotero library.

emilianoeheyns advised that the `urldate` might be missing from the bib file and I think that is the issue.  I've created a BBT debug log from one Zotero website entry; Debug log ID: EQI7MFL7-refs-euc/6.7.174-6.

Grateful for any help offered.

Sarah

**Comment by retorquere** [comment]
For me this exports to Better BibLaTeX:

```
@online{bowelcancerukItTimeTest,
  title = {It’s {{Time}} to {{Test}} for {{Lynch}} Syndrome},
  author = {{Bowel Cancer UK}},
  url = {https://www.bowelcanceruk.org.uk/campaigning/support-our-campaigns/time-to-test/},
  urldate = {2023-10-18},
  abstract = {Find out if all bowel cancer patients are being tested in your area},
  langid = {british},
  organization = {Bowel Cancer UK},
  keywords = {✅,🎯,Dissertation}
}
```

or Better BibTeX:

```
@misc{bowelcancerukItTimeTest,
  title = {It's {{Time}} to {{Test}} for {{Lynch}} Syndrome},
  author = {{Bowel Cancer UK}},
  journal = {Bowel Cancer UK},
  urldate = {2023-10-18},
  abstract = {Find out if all bowel cancer patients are being tested in your area},
  howpublished = {https://www.bowelcanceruk.org.uk/campaigning/support-our-campaigns/time-to-test/},
  langid = {british},
  keywords = {,Dissertation}
}
```

what do you see when you export this entry?

**Reply by retorquere** [reply]
In the forums, you said you got

```
@misc{bowelcanceruka,
title = {It{\textquoteright}s Time to Test for Lynch syndrome},
author = {Bowel Cancer UK, },
url = {https://www.bowelcanceruk.org.uk/campaigning/support-our-campaigns/time-to-test/},
langid = {australian}
}
```

but I can't reproduce this from `EQI7MFL7-refs-euc/6.7.174-6` (with Better or stock bib(la)tex), and the `langid` suggests it's not the same item.

**Reply by retorquere** [reply]
(I'm `emilianoeheyns` BTW)

**Comment by SarahJMills** [comment]
Thank-you so much for taking the time to look into this…

If I export the one item directly from Zotero into a .bib file, it includes all the fields (including url date) as you list below.

The issue seems to be something to do with the interface with Quarto (from RStudio/POSIT).
As in the screenshot below from https://quarto.org/docs/visual-editor/technical.html#citations, Quarto enables a direct link with my Zotero library, but the resultant .bib file it is creating is lacking the `urldate` field.

Completely understand if you say this is out of scope for you at better BibTeX…unfortunately, I tried the Quarto Github discussions but they replied the issue was out of Quarto’s scope.


￼

**Comment by retorquere** [comment]
I think if you use https://quarto.org/docs/visual-editor/technical.html#citations-from-zotero, Rstudio reads the Zotero database directly, and it generates its own bibtex, no involvement of BBT. I seem to recall it is possible to configure RStudio to use BBT, but I have no experience setting that up.

**Comment by SarahJMills** [comment]
All very helpful advice, thanks again - will follow that link and do some more tinkering.
